packages/build-essentials/src/webpack.config.js

```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2|eot|ttf|svg)$/,
+                loader: 'url-loader?limit=10000&name=./Styles/Font-[hash].[ext]&publicPath=./../'
```

Comment:
A bit ugly, but I'm not a big friend of Webpack, so maybe someone found a better solution

Reply:
I love the value of limit, it does not make sens to embed the font in the CSS; HTTP2 make a better job with concurent request, and we don't update the font a lots ;)

Reply:
ack :)